File: coding/Queue/Java/src/com/Queue/ListQueue.java
package com.Queue;
import com.LinkedList.LinkedList;

public class ListQueue {
    private LinkedList list;

    public ListQueue(){
        list = new LinkedList();
    }

    public boolean isEmpty(){
        return list.isEmpty();
    }

    public void enQueue(int value){
        list.pushFront(value);
    }

    public int deQueue(){
        int value = list.popBack();
        return  value;
    }
}

File: coding/Queue/python/ListQueue.py
from LinkedList import LinkedList

class ListQueue:
    def __init__(self):
        self.list = LinkedList()

    def isEmpty(self):
        return self.list.isEmpty()

    def enQueue(self, value):
        self.list.pushFront(value)

    def deQueue(self):
        return self.list.popBack()

